realm/realm-library/src/main/java/io/realm/internal/Util.java

```diff
@@ -48,43 +48,6 @@ public static String getTablePrefix() {
     }
     static native String nativeGetTablePrefix();
 
-
-    // Testcases run in nativeCode
-    public enum Testcase {
```

Comment:
Ok - disregard the other comment in the corresponding cpp file - I see it's not transferred :-)